winterfell/webapp/src/js/claimConfirmCtrl.js

```diff
@@ -1,23 +1,21 @@
 var app = angular.module('vetafiApp');
-app.controller('claimConfirmCtrl', ['$scope', '$location', 'net', 'claimService',
-  function($scope, $location, net, claimService) {
-    $scope.onClickBack = function() {
-      $location.path('/claim/select-forms');
-    };
+app.controller('claimConfirmCtrl', ['$scope', '$state', 'net', 'claimService', '$stateParams',
+  function($scope, $location, net, claimService, $stateParams) {
+    $scope.claimId = $stateParams.claimId;
 
     $scope.onClickConfirm = function() {
       var claim = claimService.getIncompleteClaim();
       if (!_.isEmpty(claim)) {
-        net.submitClaim(claim.id).then(function(resp) {
+        net.submitClaim($stateParams.claimId).then(function(resp) {
           // todo: set claim state or re-fetch all user claims?
           if (resp) {
-            $location.path("/claim/submitted");
+            $state.transitionTo("root.claimsubmit", {claimId: $stateParams.claimId});
           } else {
-            $location.path("/claim/submitted").search({error: 'missing'});
```

Comment:
these are in place because the template here depends on a `?error=missing` after the url.
We should probably have a better way to do this. Maybe if we're going the stateRouting way, we should have two views. One submitted success and one submitted error. There may also be different kinds of errors so the submitted error page should handle all of those.